### PR TITLE
A VecCheckNan issue 

### LINKED ISSUE #489: A VecCheckNan issue
when I enable the Nan Check env
```
 env = VecCheckNan(env, raise_exception=True)
```
the system will have a error like:

```
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/stable_baselines/ppo2/ppo2.py", line 318, in learn
    runner = Runner(env=self.env, model=self, n_steps=self.n_steps, gamma=self.gamma, lam=self.lam)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/stable_baselines/ppo2/ppo2.py", line 442, in __init__
    super().__init__(env=env, model=model, n_steps=n_steps)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/stable_baselines/common/runners.py", line 19, in __init__
    self.obs[:] = env.reset()
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/stable_baselines/common/vec_env/vec_check_nan.py", line 46, in reset
    self._check_val(async_step=False, observations=observations)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/stable_baselines/common/vec_env/vec_check_nan.py", line 58, in _check_val
    has_nan = any(np.isnan(val))
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

And I find that is cause by 2d array like following test:
```python
import numpy as np

#val = np.array([0, 0, 1, 0]) # it is run ok
val = np.array([[0, 0, 1, 0], [0, 0, 1, 0]]) # it is run same error as above
has_nan = any(np.isnan(val))
print(has_nan)
```
 and the stable-baselines at vec_check_nan like this:
```python
def _check_val(self, *, async_step, **kwargs):
        # if warn and warn once and have warned once: then stop checking
        if not self.raise_exception and self.warn_once and self._user_warned:
            return

        found = []
        for name, val in kwargs.items():
            has_nan = any(np.isnan(val))
            has_inf = self.check_inf and any(np.isinf(val))
            if has_inf:
                found.append((name, "inf"))
            if has_nan:
                found.append((name, "nan"))

```

To fix this problem, I think that need change  from 'any' to 'np.any'.

**System Info**
Describe the characteristic of your environment:
 * Describe how the library was installed (pip)
 * GPU models and configuration Mac
 * Python version 3.6.8
 * Tensorflow version 1.4.0

### PR DESCRIPTION
**Pull request by ruifeng96150**
To Fix the Bug #386 and #489 
Test pass

closes #386 
closes #489

### CODE REVIEW

**Comment by araffin** [comment]
Hello,
Thanks for the PR, I think you should also update `NanAndInfEnv` that uses `all()` instead of `np.all`.
Please also update the changelog (cf contributing guide)

File: tests/test_vec_check_nan.py [hunk 1]
```diff
@@ -68,3 +68,9 @@ def test_check_nan():
     else:
         assert False
 
+    try:
```
Comment (araffin) [inline tests/test_vec_check_nan.py]: Why put a try except if it should not fail?

File: docs/misc/changelog.rst [hunk 2]
```diff
@@ -42,6 +42,7 @@ Bug Fixes:
   `Issue #430 <https://github.com/hill-a/stable-baselines/issues/430>`_.
 - Fix a bug when calling `logger.configure()` with MPI enabled (@keshaviyengar)
 - set `allow_pickle=True` for numpy>=1.17.0 when loading expert dataset
+- Fix a bug when use VecCheckNan to check env and use numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.
```
Comment (araffin) [inline docs/misc/changelog.rst]: Please at your username + at the bottom too ;)

Comment (araffin) [inline docs/misc/changelog.rst]:
```suggestion
- Fix a bug when using VecCheckNan with numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.
```

File: tests/test_vec_check_nan.py (hunk 1 above)
Comment (ruifeng96150) [reply tests/test_vec_check_nan.py]: I think that It's normal data, I should be run successfully. Otherwise, it will make the test fail.

Comment (araffin) [reply tests/test_vec_check_nan.py]: you can remove the try/except then, if an exception occurs, the test will fail

File: docs/misc/changelog.rst (hunk 2 above)
Comment (ruifeng96150) [reply docs/misc/changelog.rst]: Thank you very much. It's my first to create a PR and I'm a little excited to do something for such a big project. :)

File: tests/test_vec_check_nan.py (hunk 1 above)
Comment (ruifeng96150) [reply tests/test_vec_check_nan.py]: ok,  as you said

**Pull request merged**